misc/write_fake_manifests.py

```diff
@@ -76,7 +76,7 @@ def path(self):
 
     def src_obj_pairs(self, path, name):
         num_sources = paretoint(55, alpha=2) + 1
-        return [(os.path.join('..', '..', path, s + '.cc'),
+        return [(os.path.join('src', path, s + '.cc'),
```

Comment:
(It's my impression that this isn't blocked on you waiting for me; is that correct?)